fabric-rendering-v1/src/main/java/net/fabricmc/fabric/api/client/rendering/v1/FabricSkyPropertyBuilder.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.rendering.v1;
+
+import java.util.Objects;
+import java.util.function.BiFunction;
+import java.util.function.BiPredicate;
+
+import com.mojang.datafixers.util.Function3;
+
+import net.minecraft.client.render.SkyProperties;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.util.math.Vec3d;
+
+public final class FabricSkyPropertyBuilder {
```

Comment:
Some javadocs here would be great, specially since this is a public API.

Reply:
I 

> 
> 
> Some javadocs here would be great, specially since this is a public API.

I tried to make a few javadocs. Could use some help with wording with some. Documentation isn't my strong suit. I had help with the other api's docs.